### DISCUSSION TITLE
死亡细胞QAQ

### DISCUSSION THREAD

**Discussion by ginhton** [Gamuxorg/bbs#373] [投递和请求游戏]
不晓得有木有哈~

**Comment by shouhuanxiaoji** [comment]
有的

**Comment by overturncat** [comment]
ubuntu 18.04运行报错
Running Dead 
Cells/tmp/.mount_Dead CSxa0dm/usr/bin//start.sh: 行 37: ./deadcells.sh: 没有那个文件或目录
Cleaning up

**Comment by wangling12** [comment]
> ubuntu 18.04运行报错
> Running Dead
> Cells/tmp/.mount_Dead CSxa0dm/usr/bin//start.sh: 行 37: ./deadcells.sh: 没有那个文件或目录
> Cleaning up

![2019-06-17 11-33-27 的屏幕截图](https://user-images.githubusercontent.com/24770600/59577347-f39c3a00-90f5-11e9-87cc-1a8ee4127de3.png)
我的打包环境就是18.04，并没有发现你的问题，试试看安装fuse是否能解决。
或者进一步写明运行方式，运行路径等。
